File: exchange/Exchange.Mobile.Core/Models/RequestModels/UploadOfferRequestModel.cs
namespace Exchange.Mobile.Core.Models.RequestModels
{
    public class UploadOfferRequestModel
    {
        public string OfferPhoto { get; set; }
        public string OfferDescription { get; set; }
        public long CategoryId { get; set; }
        public User OfferOwner { get; set; }
    }
}

File: exchange/exchange/exchangeClientApp/src/app/Models/RequestModels/upload-offer-request.model.ts
import { UserModel } from './../user.model';
import { OfferOwnerModel } from './../offer-owner.model';
export class UploadOfferRequestModel {
  offerPhoto: string;
  offerDescription: string;
  categoryId: number;
  offerOwnerDetail: OfferOwnerModel;
  userId: number;
  userEntity: UserModel;
}
